web/themes/custom/novel/templates/layout/page.html.twig

```diff
@@ -47,6 +47,7 @@
   {% include '@novel/layout/header.html.twig'
     with {
     'header': page.header,
+    'search_header': search_header
```

Comment:
Consider if there is a patttern regarding naming.

Other variables in the template seems to suggest a pattern of using `.` as a separator to scope elements.

Reply:
👍